packages/babel-preset-gatsby/src/index.js

```diff
@@ -25,16 +25,14 @@ function preset(context, options = {}) {
         node: `current`,
       }
     } else {
-      targets = {
-        browsers: pluginBabelConfig.browserslist,
```

Comment:
> Note: this will be removed in later version in favor of just setting "targets" to a query directly.

https://babeljs.io/docs/en/babel-preset-env#targetsbrowsers